ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -123,6 +126,11 @@ public ConnectSchema valueConnectSchema() {
    * @param columnName the column name, where any alias is ignored.
    * @return the column if found, else {@code Optional.empty()}.
    */
+  public Optional<Column> findColumn(final ColumnName columnName) {
+    return findColumn(columnName.name());
+  }
+
+  @VisibleForTesting
```

Comment:
does this really need to be `public` if its just for testing?  Or is this because too many tests use this?

Reply:
just `ExpressionTypeManagerTest` uses this, not sure it's actually a necessity... I'll remove this